### ISSUE TITLE
Hanging 208 response code when getting exports

### ISSUE THREAD

**Issue by paynejd** [OpenConceptLab/ocl_issues#625]
Most exports are no longer cached for staging.aws, and for most of these, when I `GET` the export, it correctly responds with a `204`. However, I am also frequently receiving `208` response codes, which should mean that the export is currently being generated. However, this is happening even though I did not do a `POST` to create the export and no matter how long I wait, the export does not get generated (even a day later).

If I do `POST` to create the export, rather than respond with a `409` (which is what it should if there was an export already being processed), it responds with a `202` and the export is successfully cached.

The bottom line is that the logic for responding with a `208` when GETting an export has a bug.

**Comment by snyaggarwal** [comment]
Fixed
